### PR TITLE
Blog post about workshops for companies

### PR DESCRIPTION
**Pull request by gvwilson**
The blog post is an edit of @jiffyclub's mail to the 'discuss' list; this PR also updates `scf/workshops.html` to be in sync with https://github.com/swcarpentry/board_pvt/pull/101.

### CODE REVIEW

File: blog/2015/04/workshops-for-companies.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  for-profit corporations. There were no objections to the idea of doing
+  workshops for corporations, but it is something new to Software Carpentry.
+  We've decided to run a pilot program encompassing five corporate workshops
+  so we can learn more about working with corporations. The pilot program
+  will allow us to gauge corporate interest and collect feedback from
+  instructors, coordinators, and others about how things go.
+</p>
+<!-- end excerpt -->
+<p>
+  During the pilot, workshops will be run on these terms:
```
Comment (jiffyclub) [inline blog/2015/04/workshops-for-companies.html]: I would simplify this line to "Workshops at corporations will be run on these terms:".

**Comment by jiffyclub** [comment]
To summarize swcarpentry/board_pvt#101 we updated the [workshop operations page](http://software-carpentry.org/scf/workshops.html) so that it does not include descriptions of how fees are split with organizing partners (that description will be transferred to the membership descriptions). We will also be updating the same page in the public board repo.

**Comment by jiffyclub** [comment]
Public board repo updated in swcarpentry/board#34.

File: blog/2015/04/workshops-for-companies.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  to reflect this new policy.
+  At the end of the pilot program the Steering Committee will consider
+  whether to continue doing workshops for for-profit entities, taking into
+  account such factors as whether the instructors, coordinators, and students
+  involved had positive experiences, and whether there seems likely to be
+  continued interest from corporations. The Steering Committee may also elect
+  to change the terms under which corporate workshops are provided.
+  (Because it has already been arranged, the upcoming workshop at Monsanto
+  will pay the lower administrative fee of $1250, but it will count toward
+  the five-workshop sample for the pilot program.)
```
Comment (wking) [inline blog/2015/04/workshops-for-companies.html]: I'd suggest explicitly covering what happens after the fifth for-a-for-profit workshop has been arranged but before the committee has decided on the new plan.  See my [discuss@ email][1] for details.

[1]: http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2015-April/002885.html

**Pull request merged**